index.js

```diff
@@ -61,6 +62,13 @@ const init = async () => {
     delete require.cache[require.resolve(`./events/${file}`)];
   });
 
+  const backgroundFiles = await readdir("./background/");
+  client.login("log", `Loading a total of ${evtFiles.length} background workers.`);
```

Comment:
client.login()?
https://wiki.teamfortress.com/w/images/7/7f/Engineer_autodejectedtie01.wav

Reply:
He is console.logging stuff in an example background worker file, however you can use the init function to do more than that if i understood correctly